aea/common.py

```diff
@@ -18,4 +18,9 @@
 # ------------------------------------------------------------------------------
 """This module contains the common types and interfaces used in the aea framework."""
 
+from typing import Any, Dict
+
+
 Address = str
+
+JSONLike = Dict[str, Any]  # temporary placeholder
```

Comment:
See here for an example to type this more carefully: https://github.com/python/mypy/issues/731#issuecomment-539905783